README.md

```diff
@@ -1,7 +1,46 @@
 # hasura-bot :robot:
 
-Hasura's own GitHub bot.
+Hasura's own GitHub bot, currently tasked with commenting on pull requests when they are opened/closed/merged.  
+Deployed on [Zeit Now](https://zeit.co/now): https://hasura-github-bot.now.sh
 
-Currently tasked with commenting on pull requests when they are opened/closed/merged.
+## Local Setup
 
-Deployed on [Zeit Now](https://zeit.co/now): https://hasura-github-bot.now.sh
+- Fork this repo and clone it on your system:
+  ```bash
+  git clone https://github.com/<username>/github-bot
+  ```
+- Navigate into the repo and install the dependencies:
+  ```bash
+  cd github-bot
+  npm install
+  ```
+- Create a `.env` file and set `GITHUB_TOKEN` and `WEBHOOK_SECRET` env variables.
+- Start a local server using `vercel`:
+  ```bash
+  npm run dev
+  ```
+  **Note:** If you have installed the `vercel` npm package globally(with `-g` flag), you can directly run `vercel dev` command.  
```

Comment:
We would typically want the developer to avoid using `vercel` directly like mentioned in this line because they might possibly run into errors due to newer versions of vercel package. More on this below:

While doing `npm install -g vercel`, the latest version of vercel package is installed in the local system. So, if a newer vercel package with breaking changes is released as latest, the user who is trying to run `vercel` command might face an error or behaviour difference of the command.

In order to prevent this, we are making use of [devDependencies](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#devdependencies) of npm to lock the version of vercel command that will get run when doing local development (via `npm run dev`). This also allows the entire dev setup to be self-contained and not rely on global packages.

Reply:
Cool, I'll remove the `vercel dev` command altogether